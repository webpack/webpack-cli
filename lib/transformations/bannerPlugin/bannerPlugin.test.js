const defineTest = require('../defineTest');

defineTest(__dirname, 'bannerPlugin', 'bannerPlugin-0');
defineTest(__dirname, 'bannerPlugin', 'bannerPlugin-1');
defineTest(__dirname, 'bannerPlugin', 'bannerPlugin-2');
