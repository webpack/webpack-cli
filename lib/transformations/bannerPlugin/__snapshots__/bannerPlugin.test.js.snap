exports[`bannerPlugin transforms correctly 1`] = `
"module.exports = {
    plugins: [
        new webpack.BannerPlugin({
            raw: true,
            entryOnly: true,
            \'banner\': \'Banner\'
        })
    ]
}
"
`;
